### PR TITLE
refactor(shell-panel): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1500

### CODE REVIEW

**Comment by jrowlingson** [comment]
It would be cool if we could set the width vars in terms of `rem` units.

File: src/components/calcite-shell-panel/calcite-shell-panel.scss [hunk 1]
```diff
@@ -1,30 +1,27 @@
...
-  align-items: stretch;
-  align-self: stretch;
-  background-color: var(--calcite-ui-background);
+  @apply items-stretch 
+    self-stretch 
+    bg-background 
+    flex 
+    p-0 
+    pointer-events-auto;
   flex-flow: column nowrap;
```
Comment (jrowlingson) [inline src/components/calcite-shell-panel/calcite-shell-panel.scss:24]: `flex-col flex-no-wrap`?

Comment (julio8a) [reply src/components/calcite-shell-panel/calcite-shell-panel.scss]: done! thx!

**Comment by julio8a** [comment]
I think this one is good to go?

**Review by jrowlingson — APPROVED** [review]

**Pull request merged**